kubernetes.go

```diff
@@ -27,21 +27,29 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/pkg/api"
 	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/pkg/runtime"
+	"k8s.io/client-go/pkg/util/wait"
 	"k8s.io/client-go/pkg/watch"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 )
 
 var defaultPortMode = "http"
+var reconnectTimeoutSeconds = 300
 
 func init() {
 	flag.StringVar(&defaultPortMode, "default-port-mode", defaultPortMode, "Default mode for service ports")
+	flag.IntVar(&reconnectTimeoutSeconds, "reconnect-timeout", 300, "Reconnect timeout in seconds")
```

Comment:
Default value for this variable could be set to reconnectTimeoutSeconds var (as defaultPortMode).